packages/waggle/src/libp2p/connectionsManager.test.ts

```diff
@@ -16,25 +15,6 @@ beforeEach(() => {
 })
 
 describe('Connections manager', () => {
```

Comment:
Maybe add some info that those tests runs without tor? (not using tor)

Reply:
I think that's self explanatory if we have `.tor.test.ts` files with tor tests. We would have to add "(not using tor)" to 95% of our test cases.
I added (using tor) to tor-test names instead.